feat(tianmu): support select into outfile with enclosed char. (#850,#1006)

lujiashun opened the pull request:
<!--

Thank you for contributing to StoneDB!

PR Title Format: <type>(<scope>): description to this pr (#issue_id)
e.g.
fix(util): fix sth..... (#1)

-->

## Summary about this PR
<!--

There MUST be one line starting with "Issue Number:  " and 
linking the relevant issues via the "close" or "ref".
e.g.:
Issue: close #1

-->

Issue Number: close #850 #1006
[summary]
1 support select into out-file can output enclose char(optionally or not); 
2 support load with enclose char(optionally or not);

## Tests Check List
<!-- At least one of next options must be included. -->

- [ ] Unit test
- [X] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

## Changelog
<!-- At least one of next options must be included. -->

- [X] New Feature
- [ ] Bug Fix
- [ ] Performance Improvement
- [ ] Build/Testing/CI/CD
- [ ] Documentation
- [ ] Not for changelog (changelog entry is not required)

## Documentation
<!-- At least one of next options must be included. -->

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features

RingsC commented on storage/tianmu/loader/parsing_strategy.cpp:
```diff
@@ -360,15 +360,27 @@ ParsingStrategy::ParseResult ParsingStrategy::GetOneRow(const char *const buf, s
+    size_t val_len{0};
+
+    if (string_qualifier_ && *val_beg == string_qualifier_) {
+      val_start = const_cast<char *>(val_beg) + 1;
```
Would you like to add some comments on these const value, 1, 2 in next pr.

lujiashun replied:
OK, I will add  some comments on these const value, 1, 2, Thanks.

Commit:
feat(tianmu): support select into outfile with enclosed char. (#850,#1006)

[summary]
1 support select into out-file can output enclose char(optionally or not);
2 support load with enclose char(optionally or not);
